cms/io/web_service.py

```diff
@@ -27,7 +27,8 @@
     import tornado.wsgi as tornado_wsgi
 from gevent.pywsgi import WSGIServer
 from werkzeug.contrib.fixers import ProxyFix
-from werkzeug.wsgi import DispatcherMiddleware, SharedDataMiddleware
+from werkzeug.middleware.dispatcher import DispatcherMiddleware
+from werkzeug.middleware.shared_data import SharedDataMiddleware
```

Comment:
Unrelated to this commit?

Reply:
Moved to `821e1b2b474d885f910f519101d06f6b97133576`